documentation/1.9/content/concepts/archive.md

```diff
@@ -68,6 +68,9 @@ The root directory under which JARs/directories used for server classpaths are s
 #### `wlsdeploy/coherence`
 The root directory under which empty directories must exist for Coherence persistent stores.
 
+#### `wlsdeploy/custom`
+This is the root directory where your custom files can be stored in and extracted from the archive. These files are not collected by the Discover tool.
```

Comment:
Minor suggestion:

tool. --> tool and are located in `<domain-home>/wlsdeploy/custom` after a model is deployed.